geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ClientProtocolService.java

```diff
@@ -32,10 +31,8 @@
    * The pipeline MUST use an available authenticator for authentication of all operations once the
```

Comment:
This should probably be processor (and in the next comment).

Reply:
fixed